paris_rollins.py

```diff
@@ -25,21 +39,42 @@
 WORKER_NICE = 19
 # paris-traceroute should take no longer than this to complete (timed out,
 # partial results will be discarded).
-WORKER_TIMEOUT = 20 
+WORKER_TIMEOUT = 60 
 # Maximum number of paris-traceoutes to run simultaneously (requests to run
 # more will be discarded).
 MAX_WORKERS = 10
+# Whether a TCP connection is closed.
+WEB100_STATE_CLOSED = 1
+# Whether a TCP connection was over IPv4
+WEB100_IPV4 = 1
+# Base source port to use when running traceroute
+PARIS_TRACEROUTE_SOURCE_PORT_BASE = 33457
+# Where to log traceroutes
```

Comment:
Re-commenting because this seems to get suppressed in the conversation view: We can delete `LOG_PATH` as it's no longer used.